cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/TranslationContext.kt

```diff
@@ -45,5 +47,8 @@ class TranslationContext(
      * The type manager is responsible for managing type information. Currently, we have one
      * instance of a [TypeManager] for the overall [TranslationResult].
      */
-    val typeManager: TypeManager
+    val typeManager: TypeManager,
+
+    // TODO(oxisto): merge to statistics holder
+    val typeObserverInvocations: AtomicInteger = AtomicInteger(0)
```

Comment:
What should this be used for? I think it was a way to exit loops of type propagations

Reply:
We can probably remove it